lib/api/request/kuzzleRequest.ts

```diff
@@ -59,7 +60,7 @@ export class KuzzleRequest {
    */
   public id: string;
 
-  constructor (data: any, options: any) {
+  constructor(data: any, options: any) {
```

Comment:
Your formatter should not delete this space. It allows to quickly find a function definition in a codebase by searching `functionName (`